h2/src/docsrc/html/migration-to-v2.html

```diff
@@ -73,4 +76,17 @@ <h2 id="general">General</h2>
 Some old non-standard SQL syntax support has been removed.
 </p>
 
+<p>
+Migrating an old database to the new version works most of the times. However, there are a couple of important changes in the new version to keep in mind:
+</p>
+
+<ul>
+<li>Restricted maximum allowed length limits: there are new length limits for for non-LOB data types. For example, for the VARBINARY type the maximum allowed size is now 1048576 instead of the previous 2^31-1. Adjust the tables before migration.
+</li><li>Oracle-style units were never supported officially, altough some worked before. For example, the length of the VARCHAR datatype cannot be more specified using CHAR but CHARACTERS or OCTETS.
```

Comment:
`CHAR` and `BYTE` can still be used in Oracle compatibility mode.

Reply:
Corrected